docs/source/github_config_keys.rst

```diff
@@ -2,15 +2,16 @@
 
  .. note::
 
-    In order to use custom GitHub repository URL, you need to set 3 additional, local git keys:
+        git-machete will try to infer the GitHub repository to fire GitHub API requests against from `git remote` data.
```

Comment:
Is the indent correct now? 🤔

Reply:
No, I think you removed part of the indent, its fixed now.

Reply:
Also i changed `git-machete will try to infer the GitHub repository to fire GitHub API requests against **from** `git remote` data.` into `git-machete will try to infer the GitHub repository to fire GitHub API requests agains `git remote` data.`

Reply:
Or even clearer:

```
git-machete will use ``git remote`` data to infer the GitHub repository to fire GitHub API requests against.